benchmark/util/text-decoder.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  len: [1, 64, 1024],
+  n: [1e6]
+});
+
+function main({ len, n }) {
+  const buf = Buffer.alloc(len, 42);
```

Comment:
Let's try a few more variations on the text encoding and content.

Reply:
Updated the benchmark to include different encodings.